react-client/src/pages/plan/research-outputs/researchoutputs.js

```diff
@@ -232,19 +236,29 @@ function ResearchOutputs() {
 
             {dmp?.dataset?.items ? dmp.dataset.items.map((item, index) => (
               <Fragment key={index}>
-                <div data-colname="name">{item.title}</div>
+                <div data-colname="name" id={"Output-" + index}  >{item.title}</div>
                 <div data-colname="personal">{item.personal}</div>
                 <div data-colname="sensitive">{item.sensitive}</div>
                 <div data-colname="repo">{item.repository.title}</div>
                 <div data-colname="datatype">{item.typeDisplay}</div>
                 <div data-colname="actions" className="form-actions">
-                  <button value={index} onClick={handleModalOpen}>
+
+                  <button
+                    id={"editOutput-" + index}
+                    aria-labelledby={"editOutput-" + index + " " + "Output-" + index}
+                    value={index}
+                    onClick={handleModalOpen}>
                     Edit
                   </button>
 
-                  <button value={index} onClick={handleDeleteOutput}>
+                  <button
+                    id={"editOutput-" + index}
+                    aria-labelledby={"editOutput-" + index + " " + "Output-" + index}
```

Comment:
Since his is a delete button, shouldn't the aria labelledby change? Looks like a copy from the other button, but you forgot to change the values?